src/literals.py

```diff
@@ -37,6 +37,7 @@
 
 S3_REL_NAME = "s3-credentials"
 S3_BACKUPS_PATH = "zookeeper_backups"
+S3_BACKUPS_LIMIT = 20
```

Comment:
**question:** What does this limit pertain to? The max number of stored backups?
How is a user meant to clean these up, manually?

Reply:
This is the maximum number of backups to be displayed with the `list-backups` action. `20` is a complete arbitrary choice, there is no meaning behind it. I figured that users are more likely to care about the latest backups than the older ones.
The reason why we limit is outlined in the method in the backup manager. As it is, we will make at most 21 queries to S3